tests/testthat/test_13_collab.R

```diff
@@ -0,0 +1,45 @@
+context("Managing collaborations")
+
+test_that("creating a collaboration works", {
+  
+  skip_if_no_token()
+  
+  writeLines("collab test file", file.path("test_dir", "collab.txt"))
+ 
+  # upload them
+  file <- box_ul(0, "test_dir/collab.txt")
+  folder <- box_dir_create("collab")
+  
+  # create collabs with the boxr tester account
+  boxr_tester_acct <- 9459307839
+  collab_file <- box_create_collab_file(file$id, boxr_tester_acct)
```

Comment:
These might make more sense as a part of the test block.

Reply:
These value get reused in other tests, so I left them outside